src/main/java/frc/robot/Util.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+//
+package frc.robot;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/** Add your docs here. */
+public class Util {
+
+    public static Translation3d getNormalTranslation(Translation3d vec){
+        return(new Translation3d(vec.getX()/vec.getNorm(), vec.getY()/vec.getNorm(), vec.getZ()/vec.getNorm()));
+    }
+
+    public static class PIDConstants{
+        public double mKP;
```

Comment:
It's generally not a great idea to expose a class' fields like this, does code outside of this class need access to these?

Reply:
Fixed this issue and added accessor methods.